stdlib/date/date.go

```diff
@@ -3,6 +3,7 @@ package date
 import (
 	"context"
 	"fmt"
+	"github.com/influxdata/flux/lang/execdeps"
```

Comment:
_nit_ We normally put all the stdlib imports first, followed by a blank line, followed by all the other imports. Each section (stdlib and the other imports) should be sorted alphabetically. Most IDEs get this wrong unfortunately.